package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@clearfeed-ai/html-to-mrkdwn-ts",
   "description": "Fast HTML to Slack flavored mrkdwn",
-  "version": "1.1.1",
+  "version": "2.0.0",
```

Comment:
Updated the package's next major version because this contains some backward incompatible changes because now we are always using some defaults for `NodeHtmlMarkdownOptions`